### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by yoshi-1012** [drand/tlock#82] [Q&A]
Do you have documentation explaining the code if I want to contribute to the project?

**Comment by CluEleSsUK** [comment]
We're a little light on the documentation of the code, but some good places to start are:
- https://drand.love/docs/timelock-encryption/
- https://eprint.iacr.org/2023/189

We released a series of code walkthrough sessions of the main drand codebase, though they don't touch on timelock encryption: https://www.youtube.com/watch?v=59WB93uqvXU&list=PLsg1px0GrGggx63o9P9_IId9T9itUYmUj
